cluster/gce/gci/health-monitor.sh

```diff
@@ -26,7 +26,19 @@ set -o pipefail
 # automatically restart the process.
 function docker_monitoring {
   while [ 1 ]; do
-    if ! timeout 60 docker ps > /dev/null; then
+    n=0
+    dockerpscheck=1
+    until [ $n -ge 10 ]; do
+      if ! timeout 60 docker ps > /dev/null; then
```

Comment:
We can probably tweak the retry times. We are not entirely sure how many retries it happened in our environment, but simply waiting for 60s has made docker being killed innocently.

`docker ps` becomes very unresponsive under certain conditions. This PR surely is merely a workaround.